src/tunnel/server/server.rs

```diff
@@ -142,6 +156,13 @@ impl WsServer {
             }
         };
 
+        self.metrics.connections.add(
+            1,
+            &[
+                KeyValue::new("remote_host", format!("{}", remote.host)),
```

Comment:
We should avoid setting remote_host and port as label as it will make too much cardinality for prometheus, and create a data leak for us.

If a user request random host / port our metrics are going to fill the available memory of wstunnel until it OOM as they are never cleaned.

metrics should not jeopardize the stability of the system
https://stackoverflow.com/a/69167162

Reply:
I'll have to think about this a bit.

I see why you say that, and agree with the point you make.
OTOH, I need the cardinality to get the information I need. So I'll have to think about some way to customize this.
In my case, k8s and multiple replicas make an OOM less serious.